armory/launcher/execute_template.py

```diff
@@ -1,7 +1,20 @@
 """Execution Script for Armory Runs
 """
+
+
+# launcher.py substitutes all names with a dollar-sign sigil to derive a runnable file.
+# Prevent execution if someone tries to run this file without substitution.
+if "$template_sentinel":
+    raise ImportError("execute_template.py is a template cannot be run directly")
+
+# TODO: since armory_sys_path is either "" or "/armory_src", and the latter only
```

Comment:
yeah I guess that makes sense since inserting `""` into the path doesn't do anything.

Reply:
Because `PATH=":/usr/bin" has a null component at the front it is a synonym for `.` (cwd). 

So too does sys.path treat a null component as cwd, so I can't be so cavalier.  I just [looked it up, so FYI.](https://docs.python.org/3/library/sys.html#sys.path)